flink-libraries/flink-state-processing-api/src/test/java/org/apache/flink/state/api/SavepointWriterUidModificationITCase.java

```diff
@@ -184,21 +214,57 @@ private static String modifySavepoint(
         return newSavepointPath;
     }
 
-    @SafeVarargs
     private static void runAndValidate(
-            String savepointPath, Tuple2<Collection<Integer>, String>... assertions)
-            throws Exception {
+            String savepointPath, ValidationParameters... validationParameters) throws Exception {
+        // validate metadata
```

Comment:
The intention here is to make sure that `UID` and `hash` are consistent with each other.